python/planout/experiment.py

```diff
@@ -315,5 +318,8 @@ def assign(self, params, **kwargs):
     def checksum(self):
         # self.script must be a dictionary
         assert hasattr(self, 'script') and type(self.script) == dict
+        src = json.dumps(self.script)
 
-        return hashlib.sha1(json.dumps(self.script)).hexdigest()[:8]
+        if not isinstance(src, six.binary_type):
+            src = src.encode("ascii")
```

Comment:
Will `src.encode("ascii")` do the same thing as `json.dumps()`?  I am wondering if this is a breaking change (i.e., if somebody is using Python PlanOut in production, will the checksums change?).